starter-code/src/viking.js

```diff
@@ -1,11 +1,76 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+   this.health = health;
+   this.strength = strength;
+}
+Soldier.prototype.attack = function () {
+    return this.strength;
+}
+Soldier.prototype.receiveDamage = function (damage) {
+    this.health -= damage;
+}
+//console.log(Soldier.receiveDamage(50));
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+    Soldier.call(this, health, strength);
+    this.name = name;
+}
+Viking.prototype = Object.create(Soldier.prototype);
 
+Viking.prototype.receiveDamage = function (damage) {
+    this.damage=damage;
+    this.health -= this.damage;
```

Comment:
Ici, pas besoin de créer une propriété this.damage pour le damage reçu, il suffit de passer ton argument "damage", comme tu as fait pour le Soldier, et ça fonctionne :)

```javascript
Viking.prototype.receiveDamage = function( theDamage ) {
    this.health -= theDamage;
    if( this.health > 0 ) {
        return this.name + " has received " + theDamage + " points of damage";
    }
    else {
        return this.name + " has died in act of combat";
    }
};
```